README.md

```diff
@@ -29,17 +29,27 @@ sudo make install
 
 ### Using:
 
-To run a container that you received just execute `runc` with the JSON format as the argument or have a
-`config.json` file in the current working directory.
+To run a container that you received just execute `runc`:
```

Comment:
This could maybe be “To run a container, execute `runc start` in the bundle's root directory:”, since [we don't want just `runc`][1].

[1]: https://github.com/opencontainers/runc/pull/210#discussion_r37682043